### ISSUE TITLE
Can't get Amarra Mantle to craft in Direwolf20 1.16 pack.

### ISSUE THREAD

**Issue by Shadowfire86** [HellFirePvP/AstralSorcery#1622]
I try crafting the mantle, I have the Attuned Crystal in the irridescent altar, the base materials too. I right click and it starts for a spllit second and then stops. The table says I have enough light and to be sure I have 2 enhanced Celestial Collector Crystals focused on it too, but it still won't craft. Have 4 others I'm going to try and will update if necessary. Same problem with Bootes and Lucerna.

TLDR: after more testing, the bug is that the indicators that you CAN craft the item trigger at far lower starlight levels than are actually required.

**Comment by PocketfulOfGeese** [comment]
shouldn't be far, should be barely on the edge of it. It's an issue with client/server world tick delays for syncing. Might be fixed in a soon version.

But yes, it's an "amount of starlight" issue.

**Comment by HellFirePvP** [comment]
The amount of starlight shown is now more consistent and should display a more accurate value of how much starlight is actually in the altar for crafting.
